src/app/components/FilterNetwork/FilterNetwork.test.tsx

```diff
@@ -175,7 +177,7 @@ describe("FilterNetworks", () => {
 
 		userEvent.click(within(screen.getAllByTestId("FilterNetwork")[0]).getByTestId("network__viewall"));
 
-		await screen.findAllByTestId("FilterNetwork__select-all-checkbox");
+		await expect(screen.findAllByTestId("FilterNetwork__select-all-checkbox")).resolves.toBeArray();
 
 		userEvent.click(screen.getAllByTestId("FilterNetwork__select-all-checkbox")[0]);
```

Comment:
```suggestion
		const selectAllCheckbox = screen.getByTestId("FilterNetwork__select-all-checkbox");

		expect(selectAllCheckbox).toBeVisible();

		userEvent.click(selectAllCheckbox);
		
		// Or in one line
		userEvent.click(screen.getByTestId("FilterNetwork__select-all-checkbox"));
```
according to [this line](https://github.com/PayvoHQ/wallet/pull/630/files#diff-c2a706c56998786118378438c58faf718b6382a6bb479caa9de1714740cd3e46R155), seems to be only one element with this test id, so i think `toBeArray` (& `jest-extended`) is unnecessary here